config-model/src/main/java/com/yahoo/vespa/model/admin/Configserver.java

```diff
@@ -26,11 +18,13 @@
  */
 public class Configserver extends AbstractService {
     private static final long serialVersionUID = 1L;
-    private static final int defaultPort = 19070;
-    private static final Logger log = Logger.getLogger(Configserver.class.getName());
+    public static final int defaultRpcPort = 19070;
```

Comment:
maybe we should change this to something from getDefaults(),
will come back to that